build_web.sh

```diff
@@ -54,7 +55,7 @@ TARGET_NAME="${CRATE_NAME_SNAKE_CASE}.wasm"
 WASM_PATH="${TARGET}/wasm32-unknown-unknown/${BUILD}/${TARGET_NAME}"
 wasm-bindgen "${WASM_PATH}" --out-dir docs --no-modules --no-typescript
 
-if [[ "${FAST}" == false ]]; then
```

Comment:
This is a misunderstanding: `--fast` makes the _building_ fast, not the output! 

Perhaps `--optimize` would be a clearer name

Reply:
Good idea - I've changed that now.